python-ufonormalizer/PKGBUILD

```diff
@@ -4,14 +4,16 @@
 pkgname=python-ufonormalizer
 _pyname=${pkgname#python-}
 pkgver=0.6.1
-pkgrel=3
+pkgrel=4
 pkgdesc='A tool that will normalize XML and other data inside of a UFO'
 arch=(any)
 url="https://github.com/unified-font-object/ufoNormalizer"
-license=(BSD)
-depends=(python)
+license=(BSD-3-Clause)
+depends=(python
+         python-setuptools)
```

Comment:
I just looked and I don't see any reason to make this a runtime dependency. Where are you getting this from?